install/build_from_source.md

```diff
@@ -114,13 +114,21 @@ For ROOT <= 6.20, an older version of PyROOT (not based on Cppyy) will be built.
 
 ## Setting the C++ standard
 
-> TODO: explain how ROOT features/STL backports interact with C++ standards and how to pick a standard
+ROOT needs to be configured and built with the same C++ standard as the programs that will make use of it.
+The relevant cmake flag is [`CMAKE_CXX_STANDARD`](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html){:target="_blank"}.
+For example, from the command line, the standard can be selected by passing one of `-DCMAKE_CXX_STANDARD={11,14,17,...}`.
+
+#### ROOT STL backports
+
+ROOT backports certain useful C++ standard library features to make them available in C++11, for example `std::make_unique` and `std::string_view`.
+The backports can be found [here in the reference guide](https://root.cern.ch/doc/master/dir_7780993579c9aa6baf9598fd7cc29d54.html).
```

Comment:
AFAIK that URL is not stable :-( Or is it? Could you check for v616 vs master?

Reply:
It happens to work if you substitute `master` with any of `v616`,`v618`,`v618`,`v620`,`v622`.

If there is a better way to link to pages in our reference or a better list of our backports we should use that though.